operations/observability/mixins/workspace/dashboards/components/ws-manager-mk2.json

```diff
@@ -3485,7 +3485,7 @@
       }
     }
   ],
-  "refresh": "5s",
+  "refresh": "30s",
```

Comment:
accidentaly changed these values in a previous PR, reverting them back to their defaults